contracts/token/ERC20/extensions/ERC1363.sol

```diff
@@ -30,16 +30,23 @@ abstract contract ERC1363 is ERC20, ERC165, IERC1363 {
 
     /**
      * @dev Indicates a failure within the {transfer} part of a transferAndCall operation.
+     * @param receiver The address to which tokens are being transferred.
+     * @param value The amount of tokens to be transferred.
```

Comment:
we are not including `The` in param natspec comments, just directly the description like [here](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/06449fe7bcfaafe43abdfc2b28fb6bb189e6ade5/contracts/interfaces/draft-IERC6093.sol#L33).

```suggestion
     * @param value Amount of tokens to be transferred.
```

Applies for the other proposed natspec param comments.